configure

```diff
@@ -494,7 +496,7 @@ def compiler_version():
   try:
     proc = subprocess.Popen(shlex.split(CC) + ['--version'],
                             stdout=subprocess.PIPE)
-  except WindowsError:
+  except OSError:
```

Comment:
According to http://stackoverflow.com/questions/8890010/os-rename-on-windows-raise-windowserror-instead-of-oserror, WindowsError is a subclass of OSError, so it will be caught by the except.